test/Table_Tests/src/Excel_Spec.enso

```diff
@@ -201,11 +260,50 @@ spec =
     Test.group "Problems" <|
         Test.specify "should handle non-existing file gracefully" <|
             bad_file = Enso_Project.data / "DoesNotExists.xlsx"
-            bad_file.read (File_Format.Excel (Range "Sheet1!A:C")) . should_fail_with File.File_Not_Found
+            bad_file.read (File_Format.Excel (Cell_Range "Sheet1!A:C")) . should_fail_with File.File_Not_Found
 
         Test.specify "should handle wrong xls_format gracefully" <|
-            xlsx_sheet.read (File_Format.Excel (Range "Sheet1!A:C") True) . should_fail_with File.Io_Error
-            xls_sheet.read (File_Format.Excel (Range "Sheet1!A:C") False) . should_fail_with File.Io_Error
-
+            xlsx_sheet.read (File_Format.Excel (Cell_Range "Sheet1!A:C") xls_format=True) . should_fail_with File.Io_Error
+            xls_sheet.read (File_Format.Excel (Cell_Range "Sheet1!A:C") xls_format=False) . should_fail_with File.Io_Error
+
+    here.spec_fmt 'XLSX reading' Examples.xlsx .read
+
+    here.spec_fmt 'XLS reading' Examples.xls .read
+
+    Test.group "Reading single cells correctly" <|
+        file = Enso_Project.data / "RangeTests.xlsx"
+
+        check_table table col_names data =
+            table.columns.length . should_equal col_names.length
+            table.columns.map .name . should_equal col_names
+            data.each_with_index idx->values->
+                table.at (col_names.at idx) . to_vector . should_equal values
+
+        Test.specify "Simple table" <|
+            check_table (file.read (File_Format.Excel (Cell_Range "Sheet1!A1"))) ["AA", "BB"] [[1,2,3,4,5,6], ["A","B","C","D","E","F"]]
+            check_table (file.read (File_Format.Excel (Cell_Range "Sheet1!A2"))) ["A", "B"] [[1,2,3,4,5,6], ["A","B","C","D","E","F"]]
+            check_table (file.read (File_Format.Excel (Cell_Range "Sheet1!A1:A1"))) ["A"] [["AA"]]
+            check_table (file.read (File_Format.Excel (Cell_Range "Sheet1!B1"))) ["B"] [["BB", "A","B","C","D","E","F"]]
+            check_table (file.read (File_Format.Excel (Cell_Range "Sheet1!B1") True)) ["BB"] [["A","B","C","D","E","F"]]
```

Comment:
I'd make this `True` parameter named and in general would suggest we try to keep a convention to name boolean and similar parameters unless it is completely clear from the context what they stand for. Here I don't know what setting we are toggling without referring to the definition of `File_Format.Excel`